plugins/catalog-backend-module-github/src/providers/GithubMultiOrgEntityProvider.ts

```diff
@@ -530,7 +532,11 @@ export class GithubMultiOrgEntityProvider implements EntityProvider {
         client,
         query: '',
       },
-    )) as UserEntity;
+    );
+
+    if (!user) {
+      return;
```

Comment:
Hm just double checking - is it safe to just return here? Do you not need to perform the delta operation to actually perform the removal, if it was previously added but now it was decided it shouldn't be? 🤔 maybe i'm misreading

Reply:
So there's only 2 possible scenarios I can think of:
1) This user originally returns `undefined` from `userTransformer` so it was never added in the first place so nothing to remove here
2) This user did not return `undefined` from `userTransformer` during a previous read and was added -> `userTransformer` now returns `undefined` (though some changed condition in the logic of `userTransfomer` or a code change). In this scenario there's not much we can do here besides `return` since we can't map this user to any actual `UserEntity` to mutate since `userTransformer` gives us `undefined`

Either case - any weird states should technically be cleaned up via the `full` mutation assuming that is run occasionally. It would be nice if maybe we can warn when we encounter scenario 2 but we also have no way to detecting which scenario we're in